rerun_cpp/src/rerun/archetypes/bar_chart_ext.cpp

```diff
@@ -0,0 +1,126 @@
+#include "bar_chart.hpp"
+
+// #define EDIT_EXTENSION
+
+namespace rerun {
+    namespace archetypes {
+
+#ifdef EDIT_EXTENSION
+        // [CODEGEN COPY TO HEADER START]
+
+        inline BarChart(rerun::datatypes::TensorBuffer buffer) {
+            this->values = rerun::components::TensorData::one_dim(std::move(buffer));
+        }
+
+        // --------------------------------------------------------------------
+        // Implicit constructors:
+
```

Comment:
Let's leave a todo to
* #3794

this is one of its trickier usecases but also an important one